### DISCUSSION TITLE
How to increase z-index of nested elements?

### DISCUSSION THREAD

**Discussion by baranowskyi** [KingSora/OverlayScrollbars#664] [Q&A]
Hi! How can I remedy this situation? I want the scroll to not overlap the dropdown menu of the inner element.
![example](https://github.com/user-attachments/assets/8fe6a87e-7993-45c9-8313-71c41e09513a)

**Comment by KingSora** [comment]
@baranowskyi Good day :)

You have the full control about `z-index`. The library doesn't enforce an value on the `z-index` value, so its up to you
